src/main/java/com/baro/memo/application/dto/CopyTemplateCommand.java

```diff
@@ -0,0 +1,7 @@
+package com.baro.memo.application.dto;
```

Comment:
🌗
template에 더 적합해 보여요
```suggestion
package com.baro.template.application.dto;
```